### DISCUSSION TITLE
Alt text for Plates and svg files?

### DISCUSSION THREAD

**Discussion by mmpadilla15** [thegetty/quire#506] [Get Help] [answered]
Hi all! I'm using the Image Alt Text Viewer Chrome Extension to see alt text. I've gone into my figure.yml file and added alt text to all figure images and plates:
![Screen Shot 2022-09-28 at 5 05 24 PM](https://user-images.githubusercontent.com/74433007/192897466-7b83262d-afe9-4158-ba7d-27ded8f17394.png)

But for some reason it's not showing up on any of my plates on either the individual pages nor on the plates overview page:

![Screen Shot 2022-09-28 at 5 03 15 PM](https://user-images.githubusercontent.com/74433007/192897658-73a87591-eb29-4c57-ad43-feaa90c6515c.png)

Also, all the built-in graphics like next arrows and buttons and boxes to open images larger are showing a message that says "SVG missing title and description":

![Screen Shot 2022-09-28 at 5 02 11 PM](https://user-images.githubusercontent.com/74433007/192897945-b23533f2-8cb5-4a9d-aab0-b028d0254cc7.png)

How can I fix these two issues?

**Comment by Erin-Cecele** [comment, marked as answer]
@mmpadilla15 Thanks, Michelle. We are looking into this for you and will be in touch ASAP.

**Reply by mmpadilla15** [reply]
@Erin-Cecele Thank you!

**Reply by geealbers** [reply]
@mmpadilla15, I've made some adjustments to the layouts that _should_ fix these issues. With the caveat that the Chrome extension doesn't seem to be honoring `aria-hidden="true"` which when added to HTML elements, allows them to be skipped over by screen readers. I've added this attribute to the SVGs, for example, since they are decorative and or have accessibility labels already included even if those labels aren't always visible on the screen. Though the extension still shows them as missing required attributes, you can ignore those warnings.

To update your project:

1. Download [layouts.zip](https://github.com/thegetty/quire/files/9727277/layouts.zip)
2. Unzip it
3. Use it to replace the existing `themes/default/layouts/` directory in your project

**Important Note:** this is assuming you haven't made any other changes in side your `themes/default/layouts/` directory before now. If you have, those changes would need to be copied over into this new set of files.

**Comment by mmpadilla15** [comment]
@geealbers Hi again! I replaced my old layouts folder with the new one and, ignoring the "svg missing" messages, all the the images in the essays look good and the alt text on the plates is showing, but ... the top of the image is showing "empty alt attribute" while the middle of the image shows the alt text:

![Screen Shot 2022-10-06 at 3 04 14 PM](https://user-images.githubusercontent.com/74433007/194408552-f2890b80-6430-4282-8060-9862e9b2e73c.png)

Weird, huh? Can that be fixed?

**Reply by geealbers** [reply]
The "Empty alt attribute" on the image is another case of the extension ignoring `aria-hidden="true"`. And the fact that the actual alt text is in the middle of the image like that is because it's actually coming from a visually-hidden version of the image rather than the zooming one. I'm honestly not 100% sure this is the best solution, but it's the best I could figure out. This older zooming image viewer has been completely replaced by a newer one in the forthcoming version of Quire, and with that change we'll be able to provide better accessibility going forward.

**Reply by mmpadilla15** [reply]
@geealbers Good to know! I'm not super worried as I'm just glad there's alt text there!
